models/patch-review.js

```diff
@@ -0,0 +1,12 @@
+const db = require('../db/connection')
+
+exports.updatedReview = (reviewId, newVote) => {
+    return db.query(`UPDATE reviews SET votes = votes + ${newVote}
+WHERE review_id = ${reviewId}
+RETURNING *`
```

Comment:
In our SQL Statements, we do not want to use template literals, as that leaves us open to SQL injection vulnerabilities.
You can find further explanation of this in the "node Postgres" notes! 😄